go.mod

```diff
@@ -1,12 +1,62 @@
-module roob.re/reroller
+module github.com/newrelic-forks/reroller
 
-go 1.16
+go 1.20
```

Comment:
We are having issues with Go 1.20 and Alpine. We think that it is because we are compiling with `CGO_ENABLED=1` but it might be more things failing. I would stick to 1.19 for now.